### DISCUSSION TITLE
Eltako FSB12 Support

### DISCUSSION THREAD

**Discussion by UsefulVid** [mak-gitdev/HA_enoceanmqtt#129] [Request]
I have an FSB12 in my household to control 2 blinds:
![image](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/39967070/f4b95e07-b3f5-483a-82ac-a4cc6e39ab0a)
Is there any chance that this may work with this integration? According to Eltako it uses the enocean standard.

After some research I found out that FSB12 seems to work with EEP A5-3F-7F

https://wiki.fhem.de/wiki/EnOcean-FSB12-RS485-Bus-Schaltaktor-2-Kanal-Beschattungselemente-Rollladen

Should I try the EnOcean integration from HA first? They claim that it will work if an official profile is used.
There was already a discussion here:
https://github.com/mak-gitdev/HA_enoceanmqtt/discussions/99

**Comment by mak-gitdev** [comment]
Hi @UsefulVid, 

If it works with FHEM and USB300, then it should be ok here as well.
I think you can try to use it as an FSB14, so simply follow the procedure on how to use FSB14.

**Reply by UsefulVid** [reply]
Thanks for your answer. Eltako told me that FSB12 is unidirectional only so it can only receive commands as far as I understood it. Hopefully this is no problem. The USB300 is now ordered, I will keep you updated if it worked :)

**Reply by UsefulVid** [reply]
@mak-gitdev I couldn't get it to work :(
Would be great to get some help
